elm.json

```
@@ -1,7 +1,7 @@
 {
     "type": "package",
-    "name": "gampleman/elm-rosetree",
-    "summary": "Strict multiway trees aka rosetrees and a Zipper to go with them",
+    "name": "gampleman/core-extra",
+    "summary": "This is a collection of utility functions that provide additional or improved functions to the Elm standard library.",
```

Since this is supposed to be a somewhat punchy summary (that's how it appears in the elm package search settings), perhaps we should do something like:

```suggestion
    "summary": "Utility functions for an improved experience with elm/core",
```

or some such...